packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -100,6 +103,60 @@ function formatBlogPostDate(locale: string, date: Date): string {
   }
 }
 
+type AuthorMap = Record<string, Author>;
+
+async function readAuthorListFile(
```

Comment:
As we have similar code in the docs plugin, we could extract some utility function that could read from js/yaml in a generic way? 

Like `readConfigFile("blog/authors",schema)`

Reply:
In fact the YAML parser should be able to parse JSON because YAML is a superset of JSON. Should I make that refactor here?

Reply:
yes discovered that recently 🤪 

I'll do it and try to merge this PR asap.

Reply:
Maybe not entirely true? https://stackoverflow.com/questions/21584985/what-valid-json-files-are-not-valid-yaml-1-1-files

I ended up not doing it because at least it works right now